mathjax3-ts/input/tex/color/ColorConfiguration.ts

```diff
@@ -0,0 +1,376 @@
+                        .setProperties({styles: { mathcolor: color }});
+    parser.stack.env['color'] = color;
+
+    parser.Push(style);
+};
+
+
+/**
+ * Define the \textcolor macro.
+ *
+ * @param {TexParser} parser The calling parser.
+ * @param {string} name The name of the control sequence.
+ */
+ColorMethods.TextColor = function (parser: TexParser, name: string) {
+    const model = parser.GetBrackets(name, '');
+    const color = getColor(model, parser.GetArgument(name));
+    const old = parser.stack.env['color'];
+
+    parser.stack.env['color'] = color;
+    const math = ParseUtil.internalMath(parser, parser.GetArgument(name));
+
+    if (old) {
+        parser.stack.env['color'] = old;
+    } else {
+        delete parser.stack.env['color'];
+    }
+    const node = parser.configuration.nodeFactory.create('node', 'mstyle', math, { mathcolor: color });
+    parser.Push(node);
+};
+
+/**
+ * Define the \definecolor macro.
+ *
+ * @param {TexParser} parser The calling parser.
+ * @param {string} name The name of the control sequence.
+ */
+ColorMethods.DefineColor = function (parser: TexParser, name: string) {
+    const cname = parser.GetArgument(name),
+        model = parser.GetArgument(name),
+        def = parser.GetArgument(name);
+
+    COLORS.set(cname, getColor(model, def));
+};
+
+/**
+ * Produce a text box with a colored background: `\colorbox`.
+ *
+ * @param {TexParser} parser The calling parser.
+ * @param {string} name The name of the control sequence.
+ */
+ColorMethods.ColorBox = function (parser: TexParser, name: string) {
+    const cname = parser.GetArgument(name),
+        math = ParseUtil.internalMath(parser, parser.GetArgument(name));
+
+    const node = parser.configuration.nodeFactory.create('node', 'mpadded', math, {
+        mathbackground: getColor('named', cname)
+    });
+
+    NodeUtil.setProperties(node, padding());
+    parser.Push(node);
+};
+
+/**
+ * Procude a framed text box with a colored background: `\fcolorbox`.
+ *
+ * @param {TexParser} parser The calling parser.
+ * @param {string} name The name of the control sequence.
+ */
+ColorMethods.FColorBox = function (parser: TexParser, name: string) {
+    const fname = parser.GetArgument(name),
+        cname = parser.GetArgument(name),
+        math = ParseUtil.internalMath(parser, parser.GetArgument(name));
+
+    const node = parser.configuration.nodeFactory.create('node', 'mpadded', math, {
+        mathbackground: getColor('named', cname),
+        style: `border: ${ColorConfigs.border} solid ${getColor('named', fname)}`
+    });
+
+    NodeUtil.setProperties(node, padding());
+    parser.Push(node);
+};
+
+new CommandMap('color', {
+    color: 'Color',
+    textcolor: 'TextColor',
+    definecolor: 'DefineColor',
+    colorbox: 'ColorBox',
+    fcolorbox: 'FColorBox'
+}, ColorMethods);
+
+export const ColorConfiguration = Configuration.create(
+    'color', { handler: { macro: ['color'] } });
```

Comment:
Why have the other commands been removed?  Don't they have to be here in order to be defined?

Reply:
@dpvc it turns out they don't. There's something I'm not sure about since I put them I get the following warning: `Configuration textcolor not found! Omitted.` for all of them.

Removing those entries fixed the warning while keeping the package functional. Am I missing something @zorkow?

https://github.com/mathjax/mathjax-v3/blob/tex_input_typescript/mathjax3-ts/input/tex/MapHandler.ts#L91-L102